conf/application.conf

```diff
@@ -0,0 +1,400 @@
+play.modules.enabled += "play.modules.reactivemongo.ReactiveMongoHmrcModule"
+play.modules.enabled += "uk.gov.hmrc.helptosaveproxy.config.CustomWSConfigParserModule"
+play.modules.enabled += "uk.gov.hmrc.helptosaveproxy.config.HealthCheckModule"
+
+# Global request handler
+# ~~~~
+# Set to the object handler for migrated 2.3 services
+play.http.requestHandler = "play.api.http.GlobalSettingsHttpRequestHandler"
+
+# Session Timeout
+# ~~~~
+# The default session timeout for the app is 15 minutes (900seconds).
+# Updating this is the responsibility of the app - it must issue a new cookie with each request or the session will
+# timeout 15 minutes after login (regardless of user activity).
+# session.maxAge=900
+
+# Secret key
+# ~~~~~
+# The secret key is used to secure cryptographics functions.
+# If you deploy your application to several instances be sure to use the same key!
+play.crypto.secret = "s6P7A361Pc3qh954bfqLg9DEQdJAsUVEuRxAeJclwOUJAGynKAweBa4OyUVhr6Jl"
+
+# Session configuration
+# ~~~~~
+application.session.httpOnly = false
+
+application.session.secure = false
+
+# The application languages
+# ~~~~~
+application.langs = "en"
+
+# Global object class
+# ~~~~~
+# Define the Global object class for this application.
+# Default to Global in the root package.
+application.global= uk.gov.hmrc.helptosaveproxy.config.MicroserviceGlobal
+
+# Router
+# ~~~~~
+# Define the Router object to use for this application.
+# This router will be looked up first when the application is starting up,
+# so make sure this is the entry point.
+# Furthermore, it's assumed your route file is named properly.
+# So for an application router like `my.application.Router`,
+# you may need to define a router file `conf/my.application.routes`.
+# Default to Routes in the root package (and conf/routes)
+# !!!WARNING!!! DO NOT CHANGE THIS ROUTER
+application.router = prod.Routes
+
+
+# Evolutions
+# ~~~~~
+# You can disable evolutions if needed
+# evolutionplugin=disabled
+
+# Logger
+# ~~~~~
+# You can also configure logback (http://logback.qos.ch/), by providing a logger.xml file in the conf directory .
+
+# Root logger:
+logger.root = ERROR
+
+# Logger used by the framework:
+logger.play = INFO
+
+# Logger provided to your application:
+logger.application = DEBUG
+
+# Metrics plugin settings - graphite reporting is configured on a per env basis
+metrics {
+  name = ${appName}
+  rateUnit = SECONDS
+  durationUnit = SECONDS
+  showSamples = true
+  jvm = true
+  enabled = true
+}
+
+
+# Microservice specific config
+
+auditing {
+  enabled = true
+  traceRequests = true
+  consumer {
+    baseUri {
+      host = localhost
+      port = 8100
+    }
+  }
+}
+
+
+mongodb {
+  uri = "mongodb://localhost:27017/help-to-save"
+}
+
+microservice {
+  metrics {
+    graphite {
+      host = graphite
+      port = 2003
+      prefix = play.${appName}.
+      enabled = false
+    }
+  }
+
+  services {
+
+    auth {
```

Comment:
is `Auth` required for Stride authenticating the service ?

Reply:
hmmm...not sure how thats all going to work, maybe not, I'm gonna leave it in for now